specification/app/resource-manager/Microsoft.App/preview/2025-10-02-preview/ContainerAppFunctions.json

```diff
@@ -0,0 +1,327 @@
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/functions/{functionName}": {
+      "get": {
+        "tags": [
+          "AzureFunctionsOnContainerApps"
+        ],
+        "summary": "Get a specific function of a Container App from the latest Revision.",
+        "operationId": "ContainerAppFunctions_GetFunction",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "Name of the Function.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ContainerAppFunction"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Container App's function": {
+            "$ref": "./examples/ContainerAppFunctions_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerAppFunction": {
+      "description": "Container App Function.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "The geo-location where the resource lives",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "Function resource specific properties",
+          "type": "object",
+          "properties": {
+            "name": {
+              "description": "Name of the Function.",
+              "type": "string",
+              "readOnly": true
+            },
+            "invokeUrlTemplate": {
+              "description": "Invoke URL for the function.",
+              "type": "string",
+              "readOnly": true
+            },
+            "triggerType": {
+              "description": "Trigger type of the function.",
```

Comment:
should we define it as enum?

Reply:
Same, It's in response from functions host which returns this as string. It also might change over time as well, so keeping it string only.